<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCChargerSet</class>
 <widget class="QWidget" name="DCChargerSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="16,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidgetChargerSet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="widgetTermParam">
        <attribute name="title">
         <string>终端参数</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetTermParam">
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>36</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>终端ID</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PDU ID</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>充电最大电流(A)</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>充电优先等级</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>充电方式</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>充电策略</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>12/24V</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>低温使能</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电子锁使能</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电子锁类型</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>VIN使能</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>国标设置</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widgetMonitorParam">
        <attribute name="title">
         <string>监控参数</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetMonitiorParam">
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>160</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>CCU ID</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>CCU ID设置 </string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>枪头起始地址</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>直流柜功率限制(kW)</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3,1,1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="labelShowTotal">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonUp">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 16pt ; } </string>
         </property>
         <property name="text">
          <string> 上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDown">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 16pt ; } </string>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 16pt ; } </string>
         </property>
         <property name="text">
          <string> 保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidgetKeyboard</class>
   <extends>QTableWidget</extends>
   <header>TableWidgetKeyboard.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
